what4/src/What4/Interface.hs

```diff
@@ -2769,6 +2793,8 @@ instance Show InvalidRange where
 class ( IsExprBuilder sym
       , IsSymFn (SymFn sym)
       , OrdF (SymExpr sym)
+      , OrdF (BoundVar sym)
+      , OrdF (SymFnWrapper sym)
```

Comment:
I believe this constraint is redundant, as there is already a top-level `OrdF (SymFnWrapper sym)` instance.